src/releasers/node.ts

```diff
@@ -81,6 +89,7 @@ export class Node extends ReleasePR {
         changelogEntry,
         version: candidate.version,
         packageName: this.packageName,
+        contents,
```

Comment:
@bcoe I just discovered a bug here while switching from `sandbox.stub` to using a mock: it caught an unexpected extra call to get the contents for package.json. The reason is that the `PackageJson` constructor was never updated to set `this.contents` so `GitHub.getChangeSet()` has to reach back out to to get it. PR coming up.